src/main/java/io/jenkins/pluginhealth/scoring/service/UpdateCenterService.java

```diff
@@ -48,6 +49,7 @@ public UpdateCenterService(@Value("${jenkins.update.center}") String updateCente
         this.updateCenterURL = updateCenterURL;
     }
 
+    @Scheduled(cron = "0 0 0 * * Mon", zone = "UTC")
```

Comment:
I think we'd just need to change this cron expression as per our need.

to run on Thursdays (at say, 2 am): "0 0 2 * * Thu"
to run every 15mins: "* */15 * * * *"

I'm referring this doc: https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions

Reply:
That's not configuration. The problem here is that if we want to change the execution schedule, we need to release the project.

Reply:
Do you mean we need to be able to input the "execution schedule" somewhere outside of the code so that we don't need to re-release the project every time just for this one change?

Would application.properties file be suitable to store the configuration?

Reply:
I'd suggest to use an environment variable to configure it.

Reply:
I've now shifted the cron string to the application.properties file and used that variable in the code.